src/styles/components/markdown-editor-preview.styl

```diff
@@ -0,0 +1,2 @@
+.markdown-editor-preview
+  display none
```

Comment:
I'm happy to leave this and figure it out later.

At worst, we could leave the preview showing under the editor, but only if it updates when you change the markdown.